src/Microsoft.AspNetCore.Server.Kestrel/Http/SocketOutput.cs

```diff
@@ -16,6 +16,7 @@ public class SocketOutput : ISocketOutput
     {
         public const int MaxPooledWriteReqs = 1024;
 
+        private const int _maxPendingWrites = 3;
```

Comment:
That is what we had before as suggested by @lodejard. Most importantly it seems better better than no limit which is what we have in RC2.